appeals/web/src/server/views/patterns/question-page.pattern.njk

```diff
@@ -0,0 +1,51 @@
+{%- extends "app/layouts/app-two-column.layout.njk" -%}
+{%- from "govuk/components/radios/macro.njk" import govukRadios -%}
+{%- from "govuk/components/checkboxes/macro.njk" import govukCheckboxes -%}
+{%- from "govuk/components/button/macro.njk" import govukButton -%}
+{%- from "govuk/components/input/macro.njk" import govukInput -%}
+{%- from "govuk/components/fieldset/macro.njk" import govukFieldset -%}
+
+
+{%- set backLinkUrl -%}
```

Comment:
Not sure if `{%- set backLinkUrl = backLinkUrl.link -%}` would work here? You could potentially avoid this entirely by passing in `backLinkUrl.link` into the render call as just `backLinkUrl`, but not sure whether the variable would automatically propagate to the layout